packages/http/src/validator/__tests__/index.spec.ts

```diff
@@ -283,3 +283,17 @@ describe('HttpValidator', () => {
     });
   });
 });
+
+describe('validateMediaType()', () => {
+  describe('when avaiable content type does not have parameters', () => {
+    const content: IMediaTypeContent = {
+      mediaType: 'application/vnd.archa.api+json',
+    };
+
+    describe('and the requeste media type matches, but has parameter', () => {
+      const result = validator.validateMediaType([content], 'application/vnd.archa.api+json; version=1');
```

Comment:
I have bad experience in tracking errors outside `it()` functions. Anything against moving to `it()`?